_rules/sequentially-focusable-element-has-visible-focus-oj04fd.md

```diff
@@ -22,62 +22,60 @@ acknowledgments:
 
 ## Applicability
 
-The rule applies to any element which is part of [sequential focus navigation][] in an [HTML document](https://html.spec.whatwg.org/#document) with at least two elements in its [sequential focus navigation order][sequential focus navigation].
+The rule applies to any element which is part of [sequential focus navigation][] in an [HTML document](https://html.spec.whatwg.org/#document).
 
 ## Expectation
 
 For each target element, there is at least one device pixel inside the [scrolling area][] of the [viewport][] whose [HSL color value](https://www.w3.org/TR/css-color-3/#hsl-color) is different when the element is [focused][] from when it is not.
 
 ## Assumptions
 
-This rule assumes that documents with only one focusable element automatically satisfy [Success Criterion 2.4.7 Focus Visible][sc247], as stated in [Understanding Success Criterion 2.4.7: Focus Visible][usc247]: "if there is only one keyboard actionable control on the screen, the success criterion would be met".
+_There are currently no assumptions._
```

Comment:
Should the _ be removed?

Reply:
I wouldn't touch that in this PR, I am taking care aof all this in #1945